helpers/reviews.js

```diff
@@ -0,0 +1,385 @@
+    } catch (err) {
+        console.error('caught err while trying to find Company:\n', err.message)
+    }
+}   
+
+//Returns a Review obj
+const getReviewById = async (reviewId) => {
+    try {
+        let foundReview = await db(MONGO_URL, () => {
+            return Review.find({ _id: reviewId }).populate("rating user company")
+        })
+        console.log('foundReview:\n', foundReview)
+        return foundReview[0]
+    } catch (err) {
+        console.error('review does not exist:\n', err.message)
+    }
+}
+
+const addCommentToReview = async (reviewId, commentId) => {
+    try {
+        //grab existing comments from review obj
+        let review = await getReviewById(reviewId)
+        let existingComments = review.comments
+        console.log("commentsList count:", existingComments.length)
+        //add new comment to list
+        existingComments.push(commentId)
+        console.log("added comment, commentsList count:", existingComments.length)
+        //update review obj
+        let result = await db(MONGO_URL, () => {
+            return Review.findByIdAndUpdate(reviewId, {
+                comments: existingComments
+            }, { new: true })
+        })
+        if (result)
+            return Status.createSuccessResponse(200, {
+                message: "Comment successfully added to Review."
+            })
+    } catch (err) {
+        console.error('add comment to review caught error:', err.message)
+        return Status.createErrorResponse(400, err.message)
+    }
+}
+
+const getAllComments = async (reviewId) => {
+    try {
+        let review = await getReviewById(reviewId)
+        return review.comments
+    } catch (err) {
+        console.error('get all comments caught error:', err.message)
+        return Status.createErrorResponse(400, err.message) 
+    }
+}
+
+//--------------- EXPORTED FUNCTIONS ---------------
+
+//013_FEAT_CRUD-REVIEW
+    //createReview 1.0
+module.exports.createReview = async (payload) => {
+    console.log('payload:\n', payload)
+    let foundUserId = await findUserId(payload) //find user
+    console.log('foundUserId:\n', foundUserId)
+    let newRatingId = await createRatingAndSave(payload) //create Rating & add to db
+    console.log('newRatingId:\n', newRatingId)
+    let foundCompany = await findCompanyByName(payload) //create Company if not exist
+    console.log('foundCompany:\n', foundCompany)
+    //Create new Review and save
+    let newReview = Review({
+        _id: new mongoose.Types.ObjectId(),
+        salary: payload.salary,
+        content: payload.content,
+        rating: newRatingId,
+        position: payload.position,
+        user: foundUserId,
+        company: foundCompany._id,
+        upvotes: [],
+        downvotes: [],
+        comments: []
+    })
+    try {
+        let result = await db(MONGO_URL, () => {
+            return newReview.save().catch(err => {
+                console.error('caught err when trying to save to db:\n', err.message)
```

Comment:
this looks like a redundant catch. The catch block outside of the try statement does the same thing